extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php

```diff
@@ -24,7 +24,7 @@ class UserProfilePageController extends WikiaController {
 	protected $title = null;
 
 	protected $defaultAvatars = null;
-	protected $defaultAvatarPath = 'http://images.wikia.com/messaging/images/';
+	protected $defaultAvatarPath = 'https://images.wikia.com/messaging/images/';
```

Comment:
Why do we change the path to `https://images.wikia.com/messaging/images/` and to `https://images.wikia.nocookie.net/messaging/images/` in `Masthead.class.php`?

Reply:
@mech please see: ec816e50a01